src/universal/bin/apalache-mc

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+#
+# Run the APALACHE model checker
+#
+# NOTE: The primary intended use for this script is to be copied into the
+# packaged produced.
+#
+# Igor Konnov, Shon Feder 2018-2022
+
+# See https://sipb.mit.edu/doc/safe-shell/
+set -euf -o pipefail
+
+# The directory where the script is run from
+# See https://stackoverflow.com/a/246128/1187277
+SOURCE=${BASH_SOURCE[0]}
+while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symlink
+  DIR=$( cd -P "$( dirname "$SOURCE" )" >/dev/null 2>&1 && pwd )
+  SOURCE=$(readlink "$SOURCE")
+  [[ $SOURCE != /* ]] && SOURCE=$DIR/$SOURCE # if $SOURCE was a relative symlink, we need to resolve it relative to the path where the symlink file was located
+done
+DIR=$( cd -P "$( dirname "$SOURCE" )" >/dev/null 2>&1 && pwd )
+
+APALACHE_JAR=${APALACHE_JAR:-"$DIR/../lib/apalache.jar"}
+JVM_ARGS=${JVM_ARGS:-""}
+TLA_PATH=${TLA_PATH:-""}
+
+if ! test -f "$APALACHE_JAR"
+then
+    echo "ERROR: No file found at ${APALACHE_JAR}"
```

Comment:
I think we had a test for the case when the directory contained jars for two different version.

Reply:
That can’t happen anymore ! :)

Reply:
this is awesome 👍